jsk_arc2017_baxter/euslisp/lib/baxterlgv7.l

```diff
@@ -59,9 +60,17 @@
                    :left_gripper_l_finger_yaw_joint
                    :left_gripper_r_finger_roll_joint_a
                    :left_gripper_r_finger_roll_joint_b))
-    ;; translate palm end for good cylindrical grasp
-    (send self :rarm-palm-endpoint :translate #f(30 0 0) :local)
-    (send self :larm-palm-endpoint :translate #f(30 0 0) :local)
+    ;; set new palm end for good cylindrical grasp
+    (setq rarm-palm-endpoint-ik- (make-cascoords))
```

Comment:
`ik` -> `lk`?

Reply:
Oh, sorry for confusing name...
I named them to show those are used in IK and different from original URDF link (right_gripper_palm_endpoint).
But now I think I should name them `rarm-palm-end-coords-` because those are coodinates, not link.
I'll fix this.

Reply:
fixed